README.md

```diff
@@ -137,3 +139,130 @@ paths. The list of functions that use this plugin can be found in
+
+    ```js
+    import petsDao from '../db/http/<resources>-dao';
+    ```
+
+## Getting data source from the Oracle Database
+
+The following instructions show you how to connect the API to an Oracle database.
+
+1. Install [Oracle Instant Client](http://www.oracle.com/technetwork/database/database-technologies/instant-client/overview/index.html) by following [this installation guide](https://oracle.github.io/odpi/doc/installation.html). **IMPORTANT:** Download the Basic Package, not the Basic Light Package.
+
+2. Define `dataSources/oracledb` section in the `/config/default.yaml` to be like:
+
+    ```yaml
+    dataSources:
+      dataSources: ['oracledb']
+      oracledb:
+        connectString: 'DB_URL'
+        user: 'DB_USER'
+        password: 'DB_PASSWD'
+        poolMin: 4
+        poolMax: 4
+        poolIncrement: 0:
+    ```
+
+    **Options for database configuration**:
+
+    | Option | Description |
+    | ------ | ----------- |
+    | `poolMin` | The minimum number of connections a connection pool maintains, even when there is no activity to the target database. |
+    | `poolMax` | The maximum number of connections that can be open in the connection pool. |
+    | `poolIncrement` | The number of connections that are opened whenever a connection request exceeds the number of currently open connections. |
+
+    > Note: To avoid `ORA-02396: exceeded maximum idle time` and prevent deadlocks, the [best practice](https://github.com/oracle/node-oracledb/issues/928#issuecomment-398238519) is to keep `poolMin` the same as `poolMax`. Also, ensure [increasing the number of worker threads](https://github.com/oracle/node-oracledb/blob/node-oracledb-v1/doc/api.md#-82-connections-and-number-of-threads) available to node-oracledb. The thread pool size should be at least equal to the maximum number of connections and less than 128.
+
+3. If the SQL codes/queries contain intellectual property like Banner table names, put them into `src/api/v1/db/oracledb/contrib` folder and use [git-submodule](https://git-scm.com/docs/git-submodule) to manage submodules:
+
+    * Add the given repository as a submodule at `src/api/v1/db/oracledb/contrib`:
+
+        ```shell
+        $ git submodule add <contrib_repo_git_url> src/api/v1/db/oracledb/contrib
+        ```
+
+    * Fetch the submodule from the contrib repository:
+
+        ```shell
+        $ git submodule update --init
+        ```
+
+4. Copy [src/api/v1/db/oracledb/pets-dao-example.js](./src/api/v1/db/oracledb/pets-dao-example.js) to `src/api/v1/db/oracledb/<resources>-dao.js` and modify as necessary:
+
+    ```shell
+    $ cp src/api/v1/db/oracledb/pets-dao-example.js src/api/v1/db/oracledb/<resources>-dao.js
+    ```
+
+5. Make sure to use the correct path for the new DAO file at path handlers files:
+
+    ```js
+    import petsDao from '../db/oracledb/<resources>-dao';
+    ```
+
+## Getting data source from an AWS S3 bucket
```

Comment:
You only need to keep AWS section here since this API only use AWS S3 bucket as datasource.